NonlinearDriver.C

```diff
@@ -112,10 +112,10 @@ bool NonlinearDriver::solutionNorms (const TimeDomain& time,
 
   const size_t nsd = model.getNoSpaceDim();
 
-  size_t iMax[nsd];
-  double dMax[nsd];
+  std::vector<size_t> iMax(nsd);
+  std::vector<double> dMax(nsd);
```

Comment:
I think for this case, realizing `nsd` is never supposed to exceed 3, it would be easier to just use fixed array sizes here instead of the std::vector, i.e.:
```
  size_t iMax[3];
  double dMax[3];
```

Reply:
ok.